worker/httpserver/tls.go

```diff
@@ -57,34 +83,54 @@ func newTLSConfig(
 		return tlsConfig
 	}
 
-	m := autocert.Manager{
+	autoCertManager := autocert.Manager{
 		Prompt:     autocert.AcceptTOS,
 		Cache:      autocertCache,
-		HostPolicy: autocert.HostWhitelist(autocertDNSName),
+		HostPolicy: autocertHostWhitelist(autocertDNSName),
 	}
 	if autocertURL != "" {
-		m.Client = &acme.Client{
+		autoCertManager.Client = &acme.Client{
 			DirectoryURL: autocertURL,
 		}
 	}
 	certLogger := SNIGetterFunc(func(h *tls.ClientHelloInfo) (*tls.Certificate, error) {
-		logger.Infof("getting certificate for server name %q", h.ServerName)
-		return nil, nil
+		logger.Debugf("getting certificate for server name %q", h.ServerName)
```

Comment:
The tests have to be updated after this new logs (levels), I see a few failing on `worker/httpserver/cert_test.go`